### PR TITLE
Surround ctx_set_ctlog_list_file() with #ifndef OPENSSL_NO_CT

### PR DESCRIPTION
**Pull request by RJPercival**
Fixes builds configured with CT disabled.

### CODE REVIEW

**Comment by richsalz** [comment]
+1

File: apps/apps.c [hunk 1]
```diff
@@ -235,6 +235,8 @@ int ctx_set_verify_locations(SSL_CTX *ctx, const char *CAfile,
     return SSL_CTX_load_verify_locations(ctx, CAfile, CApath);
 }
 
+#ifndef OPENSSL_NO_CT
+
```
Comment (ekasper) [inline apps/apps.c]: nit: don't need a blank line after the `#ifndef` (in two places in this MR)

**Comment by ekasper** [comment]
+1

**Comment by richsalz** [comment]
thanks!

**Pull request closed without merging**